packages/vue/src/stories/getting-started/4.local-development.stories.mdx

```diff
@@ -0,0 +1,97 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import pkg from "../../../package.json";
+
+<Meta title="Setup/Local Development" />
+
+<style>{`
+  a {
+    color: #5ECE7B !important;
+  }
+
+  .alert {
+    background: #efeded;
+    padding: 1rem;
+    margin: 0.5rem 0 1rem;
+  }
+
+  .tip {
+    border-left: 5px solid #5ECE7B;
+  }
+
+	.warning {
+		border-left: 5px solid #ffc107;
+		background-color: #fff4d5;
+	}
+
+	.heading {
+    font-size: 16px;
+    --mediumdark: '#999999';
+    color: #999;
+    text-transform: uppercase;
+    font-weight: 900;
+    letter-spacing: 5px;
+		margin-top:10px;
+  }
+`}</style>
+
+# Local Development
+
+<div class="heading">
+  Version
+  <a
+    href={`?path=/story/releases-v${pkg.version.replaceAll(
+      ".",
+      "-"
+    )}-change-log--page`}
+  >
+    {pkg.version}
+  </a>
+</div>
+
+If you would like to create your own version of Storefront UI by forking the project, or contribute to the project itself you should know how to run and test Storefront UI locally.
```

Comment:
If we talk about running Storefront UI locally we should also write how to fork project from github repo. It is already mentioned in contributing guide -> working with github section so it will be enough to just link it. What do you think @Baroshem ?